features/mbedtls/targets/TARGET_CRYPTOCELL310/aes_alt.c

```diff
@@ -782,40 +800,50 @@ int mbedtls_aes_setkey_sw_enc( struct sw_aes_context *ctx, const unsigned char *
             }
             break;
     }
-    ctx->sw_key_set = 1;
+    ctx->is_sw = 1;
 
     return( 0 );
 
 }
 
 //taken from the aes.c function mbedtls_aes_setkey_dec
-int mbedtls_aes_setkey_sw_dec( struct sw_aes_context *ctx, const unsigned char *key,
+int mbedtls_aes_setkey_sw_dec( mbedtls_aes_context *ctx, const unsigned char *key,
                     unsigned int keybits )
 {
     int i, j, ret;
-    struct sw_aes_context cty;
+    mbedtls_aes_context cty;
     uint32_t *RK;
     uint32_t *SK;
 
-    memset ( &cty, 0, sizeof( struct sw_aes_context ) );
+    memset ( &cty, 0, sizeof( mbedtls_aes_context ) );
```

Comment:
OK, it's residue from the rename, because `mbedtls_aes_init` receives `mbedtls_aes_context` and not `sw_aes_context` as it was before